src/Services/JiraSyncWorkflow.php

```
@@ -0,0 +1,275 @@
+     */
+    public function push(array $user, array $integration, int $projectId, string $jiraProjectKey, string $ipAddress, string $userAgent): array
+    {
+        [, $sync] = $this->makeJiraSync($integration);
+
+        $wiResult = $sync->pushWorkItems($projectId, $jiraProjectKey);
+        $usResult = $sync->pushUserStories($projectId, $jiraProjectKey);
+        $rkResult = $sync->pushRisks($projectId, $jiraProjectKey);
+
+        Integration::updateForOrg($this->db, (int) $integration['id'], (int) ($integration['org_id'] ?? 0), [
+            'last_sync_at' => ($this->clock)(),
+        ]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\StratFlow\Models\Integration' in method 'push'. (undefined)

(StaticAccess)